dbus/methods.go

```diff
@@ -598,3 +598,43 @@ func unitPath(name string) dbus.ObjectPath {
 func unitName(dpath dbus.ObjectPath) string {
 	return pathBusUnescape(path.Base(string(dpath)))
 }
+
+type JobStatus struct {
```

Comment:
As this is a public type with public fields, it would be nice to have docstrings everywhere.

Reply:
Done